vocabularies-des/qesd-obs-props.ttl

```diff
@@ -501,6 +502,21 @@
     skos:definition "A clear, odorless, tasteless liquid that is essential for most animal and plant life, and is an excellent solvent for many substances."@en ; 
 . 
 
+<https://linked.data.gov.au/def/qld-obsprop/2ac0e547-d4ca-5167-a250-79bc59b4f452>
+   a skos:Concept ;
+    skos:topConceptOf <https://linked.data.gov.au/def/qld-obsprop> ;
+    skos:inScheme <https://linked.data.gov.au/def/qld-obsprop> ; 
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/qld-obsprop> ;
+    skos:narrower 
+         <https://linked.data.gov.au/def/qld-obsprop/5fb80b7c-b505-5960-853a-23c85c7cf472> ,
+         <http://pid.geoscience.gov.au/def/voc/ga/netcdfldtos/sea_surface_temperature> ;
```

Comment:
invalid URL

Reply:
The redirection is not working. When on the ARDC Research Vocabs Australia website, the term can be found and resolves internally.